tests/integration/kubernetes/confidential/k8s-confidential.bats

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+IMAGE_REPO="ghcr.io/confidential-containers/test-container"
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	( [ "${KATA_HYPERVISOR}" == "firecracker" ] || [ "${KATA_HYPERVISOR}" == "dragonball" ] || [ "${KATA_HYPERVISOR}" == "qemu-tdx" ] || [ "${KATA_HYPERVISOR}" == "qemu-snp" ] ) && skip "Test not supported yet."
```

Comment:
What's the final target you want?

IMHO, we should be targetting SEV / SNP / TDX.  Is that the case?

Reply:
Yes, that is the case. But I am thinking I will only add SEV test case in this PR. And once it is merged, we can open 2 more PRs for SNP and TDX. Essentially, this will be the template that we can use later.

Reply:
Seems like it would be simple to at least add SNP support to this PR. If we do this right it should be very minimal effort to add another platform.

Reply:
I've made some comments on how we can change the PR to actually made it a minimal effort to add another platform, without leaving the work to re-write it to the next one to touch it. :-)